.github/workflows/ci-format.yml

```diff
@@ -58,13 +58,18 @@ jobs:
           . ${{env.venv_dir}}/bin/activate
           pip install --upgrade pip wheel
           pip install .[camera-obs,dev,extras,opendrive,ros,test,train]
-      - name: Get changed files
+          # install the specific version we want for pytest, just in case...
+          pip install pytype==2022.1.13
```

Comment:
nit:I would prefer to not have this specific install since it disconnects the test from what is listed in `setup.py`.

```suggestion
```

Reply:
My thought here was to allow us to use a newer, possibly-more-restrictive version of `pytype` in our development and local test environments if we want, while keeping the version in CI pinned to something we know is "sane".

However,  I think you're right this might be confusing to people if, at some point, newer versions of `pytype` become _less_ restrictive for whatever reason.  Then they will see things passing locally, but not in CI, which might confuse them.  So I'll take your suggestion.